svm/src/account_loader.rs

```diff
@@ -318,14 +319,8 @@ impl<'a, CB: TransactionProcessingCallback> AccountLoader<'a, CB> {
 // In general, most accounts we load this way should already be in our accounts store.
 // Once SIMD-0186 is implemented, 100% of accounts will be.
 impl<CB: TransactionProcessingCallback> TransactionProcessingCallback for AccountLoader<'_, CB> {
-    fn get_account_shared_data(&self, pubkey: &Pubkey) -> Option<AccountSharedData> {
-        self.do_load(pubkey).0
-    }
-
-    fn account_matches_owners(&self, pubkey: &Pubkey, owners: &[Pubkey]) -> Option<usize> {
-        self.do_load(pubkey)
-            .0
-            .and_then(|account| owners.iter().position(|entry| entry == account.owner()))
+    fn get_account_shared_data(&self, pubkey: &Pubkey) -> Option<(AccountSharedData, Slot)> {
+        self.do_load(pubkey).0.map(|account| (account, 0))
```

Comment:
This is a path used in production. Would you mind explaining here why the slot does not matter?

Reply:
It will matter, but not in this PR. It is only to get the interface changes at the v3.0 version bump. So for now, the slot is a dummy. I think CI forbids us from leaving TODO comments.